ibis/omniscidb/tests/test_client.py

```diff
@@ -215,7 +194,7 @@ def test_read_csv(con, temp_table, filename):
     table.read_csv(filename, header=False, quotechar='"', delimiter=",")
 
     df_read_csv = table.execute()
-    df_expected = db.table("functional_alltypes").execute()
+    df_expected = df_alltypes
```

Comment:
thanks @dchigarev for working on that. I think you don't need to have df_expected here .. you can just compare it with df_alltypes directly.

`pd.testing.assert_frame_equal(df_alltypes, df_read_csv)`.

besides that, everything looks good to me :)